src/lua/BabblerBomb/NS2Utility_Server.lua

```diff
@@ -0,0 +1,64 @@
+local LockAbility = debug.getupvaluex(UpdateAbilityAvailability, "LockAbility")
+local UnlockAbility = debug.getupvaluex(UpdateAbilityAvailability, "UnlockAbility")
+
+function UpdateAbilityAvailability(forAlien, tierOneTechId, tierTwoTechId, tierThreeTechId)

```

Comment:
Side note: I always wanted to get rid of this global method and move it directly to some Mixin :D